start.R

```diff
@@ -297,6 +297,9 @@ modeltestRunsUsed <- 0
 
 testOneRegi_region <- ""
 
+# Save whether model is locked before runs are started
+model_was_locked <- if (file.exists(".lock")) TRUE else FALSE
```

Comment:
With the upcoming PR pik-piam/gms#23 the existence of the lock file will not be meaningful anymore. Once the PR is merged we need to change it here.

Reply:
Thanks, I will discuss with Mika!